src/ResourceManager/SiteRecovery/Commands.SiteRecovery/ProtectionEntity/StartAzureSiteRecoveryPlannedFailoverJob.cs

```diff
@@ -83,10 +83,23 @@ public class StartAzureSiteRecoveryPlannedFailoverJob : SiteRecoveryCmdletBase
         /// Gets or sets the Optimize value.
         /// </summary>
         [Parameter]
-        [ValidateSet(Constants.ForDowntime, Constants.ForSynchronization)]
+        [ValidateSet(Constants.ForDownTime, Constants.ForSynchronization)]
         public string Optimize { get; set; }
 
         /// <summary>
+        /// Gets or sets the recovery vm creation value.
+        /// </summary>
+        [Parameter]
```

Comment:
These are non-mandatory and they don't take value from pipeline. We have not added anything as by default these values are false.